client/my-sites/stats/stats-purchase/stats-purchase-wizard.jsx

```diff
@@ -1,21 +1,22 @@
 /* eslint-disable jsx-a11y/anchor-is-valid */
-import { Button, CheckboxControl, Card, Panel, PanelRow, PanelBody } from '@wordpress/components';
+import { Button, Card, Panel, PanelRow, PanelBody } from '@wordpress/components';
 import classNames from 'classnames';
 import { useTranslate } from 'i18n-calypso';
 import React, { useState } from 'react';
+import statsPurchaseBackgroundSVG from 'calypso/assets/images/stats/purchase-background.svg';
 import CommercialPurchase from './stats-purchase-commercial';
 import PersonalPurchase from './stats-purchase-personal';
 import StatsPurchaseSVG from './stats-purchase-svg';
 import './styles.scss';
 
 const COMPONENT_CLASS_NAME = 'stats-purchase-wizard';
 const SCREEN_TYPE_SELECTION = 0;
-const SCREEN_PERSONAL_CHECKLIST = 1;
-const SCREEN_PURCHASE = 2;
+const SCREEN_PURCHASE = 1;
 const TYPE_PERSONAL = 'Personal';
 const TYPE_COMMERCIAL = 'Commercial';
 const DEFAULT_STARTING_PRICE = 6;
 const FLAT_COMMERCIAL_PRICE = 10;
+const IMAGE_CELEBRATION_PRICE = 8;
```

Comment:
Could we note that we'll want to pull the pricing slider configuration values from the API in a future PR?

p1688673993777489/1688595086.573129-slack-C0438NHCLSY